packages/__examples__/.storybook/stories/generateComponentExamples.js

```diff
@@ -24,30 +24,12 @@
 const { uid } = require('@instructure/uid')
 const generatePropCombinations = require('./generatePropCombinations')
 
-module.exports = function generateComponentExamples(
-  Component,
-  config = {
-    sectionProp: null,
-    maxExamplesPerPage: null,
-    propValues: {},
-    excludeProps: [],
-    getComponentProps: (props, index) => {
-      return {}
-    },
-    getExampleProps: (props, index) => {
-      return {}
-    },
-    getParameters: (examples, pageIndex) => {
-      return {}
-    },
-    filter: (props) => false
-  }
-) {
```

Comment:
This was not needed since this method always gets a value

Reply:
To be honest, I was using this to see what options it accepts :D Well, it is available in the readme too. Do you think we should make a TS type for it so we can use that in the `xxx.examples.tsx` files?

Reply:
I've added a JSDoc about the parameters, the Readme has detailed docs